tests/test_exceptions.py

```
@@ -1145,6 +1154,51 @@ def test_return_value_correct_true(self):
+            "                inner = inside\n"
+            "        except Exception:\n"
+            "            pass\n"
+        )
+        out = decompile(src)
+        lines = out.splitlines()
+        for_idx = next((i for i, ln in enumerate(lines) if ln.strip() == "for char in inside:"), -1)
+        if for_idx != -1:
+            for_indent = len(lines[for_idx]) - len(lines[for_idx].lstrip())
+            loop_body = []
+            for ln in lines[for_idx+1:]:
+                if not ln.strip(): continue
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)